test/integration/test_conferences.py

```diff
@@ -0,0 +1,584 @@
+        The following endpoints are tested in this flow:
+            - list_conference_recordings
+            - get_conference_recording
+            - download_conference_recording
+        """
+
+        # Create a conference and have it recorded
+        answer_url = MANTECA_BASE_URL + "/bxml/joinConferencePause"
+        (test_id, call_id) = self.create_conference(answer_url)
+
+        list_conferences_response = self.conference_api_instance.list_conferences(BW_ACCOUNT_ID, _return_http_data_only=False)
+
+        self.assertEqual(list_conferences_response[1], 200)
+        conferenceId = (list_conferences_response[0][len(list_conferences_response[0])-1].id)
+        
+        # update Conference Bxml Test
+        """Validate an UpdateConferencelBxml Request to start and stop recording
+        """            
+        updateBxmlBody = '<?xml version="1.0" encoding="UTF-8"?><Bxml><StartRecording/><SpeakSentence locale="en_US" gender="female" voice="susan">This should be a conference recording.</SpeakSentence><StopRecording/></Bxml>'
+        update_conference_bxml_response = self.conference_api_instance.update_conference_bxml(BW_ACCOUNT_ID, conferenceId, updateBxmlBody, _return_http_data_only=False)
+        self.assertEqual(update_conference_bxml_response[1], 204)   
+        
+        time.sleep(10)
+
+        # List Conference Recordings Endpoint
+        list_conference_recordings_response: List[ConferenceRecordingMetadata] = self.conference_api_instance.list_conference_recordings(BW_ACCOUNT_ID, conferenceId, _return_http_data_only=False)
+        assert list_conference_recordings_response[1] == 200
+
+        # We should get back at least 1 recording
+        conference_recordings = list_conference_recordings_response[0]
+        assert len(conference_recordings) > 0
+
+        # Checks on the first recording
+        first_recording: ConferenceRecordingMetadata = conference_recordings[0]
+        self.validate_recording(first_recording, conferenceId)
+        recording_id = first_recording.recording_id
+
+        # Get Single Recording Endpoint
+        recording_response: ConferenceRecordingMetadata = self.conference_api_instance.get_conference_recording(BW_ACCOUNT_ID, conferenceId, recording_id, _return_http_data_only=False)
+        self.assertEqual(recording_response[1], 200)
+        self.assertEqual(recording_response[0].conference_id, conferenceId)
+        self.assertEqual(recording_response[0].recording_id, recording_id)
+        self.assertIs(type(recording_response[0].name), str)  
+
+        self.validate_recording(recording_response[0], conferenceId)
+
+        # Download recording media
+        recording_media_response = self.conference_api_instance.download_conference_recording(BW_ACCOUNT_ID, conferenceId, recording_id, _preload_content=False)
+        conference_recording_media = recording_media_response.data                   
+
+    # List conferences 401
```

Comment:
I dont think all of these invalid tests need a comment at the beginning - the docstring suffices